### DISCUSSION TITLE
Filename prefixes to use

### DISCUSSION THREAD

**Discussion by Martin-Jung** [iiasa/BNRTools#7] [General]
We already have some mismatching filenames to use, so I suggest a few categories here:

- `conv_*` = For conversion between model inputs (e.g. G4M 2 grided spatial)
- `spl_ *` = For any generic spatial operations (e.g. spatial aggregation). Suggesting `spl` to differentiate from `sp` 
- `misc_*` = Other generic general purpose functions

Further suggestions welcome, but let's post them here first!

**Comment by mhesselbarth** [comment]
Sounds good 👍
